manifests/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+- crd.yaml
+- namespace.yaml
+- rbac.yaml
+- deployment.yaml
+- service.yaml
+commonLabels:
+  kustomize.component: pytorch-operator
+images:
+- name: gcr.io/kubeflow-images-public/pytorch-operator
+  newName: gcr.io/kubeflow-images-public/pytorch-operator
+  newTag: vmaster-gd596e904
```

Comment:
Maybe we deploy `latest` tag for controller in pytorch-operator repo by default? 
What do you think?

Reply:
make sense. I will make the change.

Reply:
Change to `latest`